### PR TITLE
Tribute To Areeb Sir

### PR DESCRIPTION
**Pull request by Creatoon**
#7202 

#### Short description of what this resolves:
Tribute To Areeb Sir Please Merge This PR.

Screenshot_2021-05-01 Open Event

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Creatoon** [comment]
@mariobehling sir please merge this PR, very thankful to you.

**Review by sachinchauhan2889 — APPROVED** [review]

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please change it as follows:
- [x] Move text in front of "Copyright" and use the same font and size as the text here. Show it in the same line to the left of Copyright.
- [x] Change "Copyright © 2021 eventyay" to "© 2021 eventyay"

**Comment by Creatoon** [comment]
@mariobehling @sachinchauhan2889  After Changes

Screenshot_2021-05-06 eventyay

File: app/templates/components/footer-main.hbs [hunk 1]
```diff
@@ -105,7 +105,8 @@
     <div class="ui stackable grid">
       <div class="thirteen wide computer twelve wide tablet column left aligned">
         <div class="ui horizontal inverted small divided link list no top margin">
-          <span class="item">{{t 'Copyright'}} &copy; {{general-date (now) 'YYYY'}}  {{this.settings.appName}} </span>
+          <span class="item">Made with <i class="red heart icon"></i> By- Areeb Jamal & Team</span>
```
Comment (MeghalBisht) [inline app/templates/components/footer-main.hbs:108]: Please wrap in t

File: app/templates/components/footer-main.hbs [hunk 2]
```diff
@@ -105,7 +105,8 @@
     <div class="ui stackable grid">
       <div class="thirteen wide computer twelve wide tablet column left aligned">
         <div class="ui horizontal inverted small divided link list no top margin">
-          <span class="item">{{t 'Copyright'}} &copy; {{general-date (now) 'YYYY'}}  {{this.settings.appName}} </span>
+          <span class="item">{{t 'Made with'}} <i class="red heart icon"></i> {{t 'By- Areeb Jamal'}} &amp; {{t 'Team'}}</span>
```
Comment (MeghalBisht) [inline app/templates/components/footer-main.hbs:108]: Name cannot be translated

Comment (Creatoon) [reply app/templates/components/footer-main.hbs:108]: ok

**Review by sachinchauhan2889 — APPROVED** [review]

**Comment by mariobehling** [comment]
Please change the heart to the same color as the text.

File: app/templates/components/footer-main.hbs [hunk 3]
```diff
@@ -105,7 +105,8 @@
     <div class="ui stackable grid">
       <div class="thirteen wide computer twelve wide tablet column left aligned">
         <div class="ui horizontal inverted small divided link list no top margin">
-          <span class="item">{{t 'Copyright'}} &copy; {{general-date (now) 'YYYY'}}  {{this.settings.appName}} </span>
+          <span class="item">{{t 'Made with'}} <i class="heart icon"></i> {{t 'By'}}- Areeb Jamal &amp; {{t 'Team'}}</span>
```
Comment (MeghalBisht) [inline app/templates/components/footer-main.hbs:108]: We can just add "Maintained by" instead of "made with heart"

**Review by MeghalBisht — DISMISSED** [review]

**Review by ranjitkshah — APPROVED** [review]

**Pull request merged**